### PR TITLE
Register faultysden.is-a.dev

### PR DESCRIPTION
**Pull request by FaultyFaulty-glitch**
Register faultysden.is-a.dev with TXT record pointing to google-site-verification=Ix1jCzU2q87IgryC4eAEXvdv1VRzjXoLU14sjfpSnko.

### CODE REVIEW

File: domains/faultysden.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "FaultyFaulty-glitch",
+           "email": "faultywindows+github@gmail.com",
+           "discord": "1041028534690386051"
+        },
+    
+        "record": {
+            "TXT": "google-site-verification=ix1jczu2q87igryc4eaexvdv1vrzjxolu14sjfpsnko"
```
Comment (DEV-DIBSTER) [inline domains/faultysden.json:9]: TXT record alone doesn't do anything here.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/faultysden.json (hunk 1 above)
Comment (FaultyWindows) [reply domains/faultysden.json:9]: Well Google Search Console told me to put that alone.

Comment (FaultyWindows) [reply domains/faultysden.json:9]: I'm just my other github account, but the thing is Google Search Console told me that this is the only thing I need to do

Comment (21Z) [reply domains/faultysden.json:9]: why do you need to add a website to google without anything in it

Comment (FaultyWindows) [reply domains/faultysden.json:9]: Bro listen, I want to add an custom domain to my Google Sites page.
It said my domain is not verified, so I need to verifiy it with that txt record.

Comment (CuteDog5695) [reply domains/faultysden.json:9]: Can't you verify your Google Sites page via HTML or did I get that information elsewhere?

Comment (FaultyWindows) [reply domains/faultysden.json:9]: > Can't you verify your Google Sites page via HTML or did I get that information elsewhere?

Yeah, but where would I put it?

**Comment by orxngc** [comment]
Closing as stale

**Pull request closed without merging**